src/components/BlogList/BlogList.jsx

```diff
@@ -1,27 +1,43 @@
-import React from "react";
+import React, { useState } from "react";
+import PropTypes from "prop-types";
 import { useSelector } from "react-redux";
 import BlogCard from "@components/BlogCard/BlogCard";
 import NotFound from "@components/NotFound/NotFound";
+import Button from "@components/Button/Button";
 import "@components/BlogList/blogList.scss";
 
 const BlogList = ({ blogs }) => {
   const searchQuery = useSelector((state) => state.search.searchQuery);
   const filteredTag = useSelector((state) => state.filter.filteredTag);
+  const [blogsToShow, setBlogsToShow] = useState(9);
```

Comment:
put the default Blogs to Show count to a variable like `DEFAULT_BLOGS_TO_SHOW`

Reply:
done